### PR TITLE
[WIP] Adding RMS and percentile univariate feature functions

### PR DESCRIPTION
**Pull request by hubertjb**
This PR introduces two additional simple univariate feature functions, root mean square (RMS) and percentiles, in case there is interest in including them in the library.

### CODE REVIEW

**Comment by jbschiratti** [comment]
Thanks for adding those feature functions @hubertjb !
LGTM
Could you, please, update the [`whats_new.rst`](https://github.com/mne-tools/mne-features/blob/master/doc/whats_new.rst) file?

**Comment by hubertjb** [comment]
Thanks @jbschiratti! I just updated whats_new.

File: doc/whats_new.rst [hunk 1]
```diff
@@ -17,6 +17,7 @@ Enhancements
 ~~~~~~~~~~~~
 
 - Extended feature naming to all univariate functions. If no specific function is implemented for a given univariate function and that the output has the same shape as the input on the feature dimension, then a generic list of feature names is passed. Possibility to rename these feature by passing ``ch_names`` to :func:`mne_features.extract_features`. By `Paul ROUJANSKY`_ in `#60 <https://github.com/mne-tools/mne-features/pull/60>`_.
+- Added root mean square (RMS) and percentile univariate feature extraction functions. By `Hubert BANVILLE`_ in `#60 <https://github.com/mne-tools/mne-features/pull/61>`_.
```
Comment (agramfort) [inline doc/whats_new.rst:20]:
```suggestion
- Added root mean square (RMS) and percentile univariate feature extraction functions. By `Hubert Banville`_ in `#60 <https://github.com/mne-tools/mne-features/pull/61>`_.
```

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
this doc page http://mne.tools/mne-features/api.html needs to be updated. thx

File: mne_features/univariate.py [hunk 2]
```diff
@@ -257,6 +258,62 @@ def compute_kurtosis(data):
...
+
+    Returns
+    -------
+    output : ndarray, shape (n_channels * len(q),)
+
+    Notes
+    -----
+    Alias of the feature function: *prct*
+    """
+    return np.ravel(np.percentile(data, q, axis=-1))
```
Comment (agramfort) [inline mne_features/univariate.py:298]: actually I would prefer to use the word "quantile" with https://numpy.org/devdocs/reference/generated/numpy.quantile.html

I find it less ambiguous about what it does.

so compute_quantile 

wdyt?

Comment (hubertjb) [reply mne_features/univariate.py:298]: Yes, that makes a lot of sense @agramfort. I'll change `percentile` for `quantile`.

**Comment by agramfort** [comment]
thx @hubertjb

**Pull request merged**